wordpress-mobile/WordPress-Aztec-Android
# Issue/238 add underline format

### Fix
Add underline to format toolbar as described in https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/238.

### Test
1. Open demonstration app.
2. Notice ***Underline*** format button.
3. Select text.
4. Tap ***Underline*** format button.
5. Notice selected text is underline.
6. Tap ***Underline*** format button.
7. Notice selected text is not underlined.
